lib/simple_crud/simple_crud_controller.rb

```diff
@@ -0,0 +1,93 @@
+require 'byebug'
+require 'active_support/all'
+require_relative 'simple_crud_helpers'
+require_relative 'simple_crud_method_helpers'
+require_relative 'valid_method_checker'
+require_relative 'policy_checker'
+require_relative 'serializer_checker'
+module SimpleCrudController
+  cattr_accessor :params, :permitted
+
+  def simple_crud(all_parameters = {})
+    all_methods = %i[show index destroy create update]
+    methods_to_create = all_parameters[:only] || all_methods - [all_parameters[:except]]
```

Comment:
Shouldn't this be: `all_methods - [all_parameters[:except]].flatten` ?

Because `except: %i[index show]` won't work in this case.